internal/provisioner/kops_provisioner_cluster_installation_beta.go

```diff
@@ -139,6 +138,17 @@ func (provisioner *kopsCIBeta) HibernateClusterInstallation(cluster *model.Clust
 	return nil
 }
 
+func configureInstallationForHibernation(mattermost *mmv1beta1.Mattermost) {
+	// Hibernation is currently considered changing the Mattermost app deployment
+	// to 0 replicas in the pod. i.e. Scale down to no Mattermost apps running.
+	// The current way to do this is to set a negative replica count in the
+	// k8s custom resource. Custom ingress annotations are also used.
+	// TODO: enhance hibernation to include database and/or filestore.
+	mattermost.Spec.Replicas = int32Ptr(0)
+	mattermost.Spec.IngressAnnotations = getHibernatingIngressAnnotations()
+	mattermost.Spec.Size = ""
```

Comment:
Same here. Are we sure the operator isn't just inserting the default value when this is done?